lib/active_fedora/associations.rb

```diff
@@ -45,7 +45,7 @@ module Builder
       autoload :BelongsTo,           'active_fedora/associations/builder/belongs_to'
       autoload :HasMany,             'active_fedora/associations/builder/has_many'
       autoload :HasAndBelongsToMany, 'active_fedora/associations/builder/has_and_belongs_to_many'
-      autoload :Contains,            'active_fedora/associations/builder/contains'
+      autoload :BasicContainsOne,    'active_fedora/associations/builder/basic_contains_one'
```

Comment:
I'm confused by the name `BasicContainsOne`. This seems like it holds a sub-resource at a known location (which we should feel bad about doing, instead of defining and using predicates..); maybe `has_subresource` would be a better name?